contrib/helm/emissary/values.yaml

```diff
@@ -0,0 +1,84 @@
+# Default values for emissary.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: emissary
+  pullPolicy: IfNotPresent
```

Comment:
I'm going to backpedal on my own advice and leave this here as an FYI because using when using minikube, this is desirable. Once I load the image with `minikube image load emissary:latest` a pull policy of IfNotPresent or Never works. When I switch to Always it tries to reach outside to docker.io and pull which does not work.